src/views/Dashboard/Customer/OrderHistoryDetail/index.vue

```diff
@@ -409,6 +411,8 @@ export default {
       this.orderId = this.$route.params.number;
       this.order = await getOrdersData(this.api, this.orderId);
       this.orderDetail = await getOrdersDetail(this.api, this.orderId);
+      console.log(this.order, 'this.order')
```

Comment:
Do you need this? Maybe we need to make a Debug mode that allows `console.log`? cc: @kevinjanada

Reply:
Making a Debug mode is **optional** and is just a suggestion.